blog/_posts/2018-11-18-TeamSectumsempraGHCI.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Team Sectumsempra at GHCI 2018!
+layout: post
+created_at: Tue Dec 18 2018
+permalink: blog/2018-11-18-TeamSectumsempraGHCI
+current: blog
+author: Avneet Kaur
+twitter: TeamSectumsem18
+categories:
+ - student-posts
+---
+
```

Comment:
I've noticed a couple of small punctuation issues and typos, so if that's ok with you I will be making changes to that separately after merging the PR. It's mainly missing spaces and such, no big deal :)
I also noticed that there are very few paragraphs, which makes it really hard to read. So if that's ok, I will be adding those (separating the text into smaller “blocks” so that it becomes easier to read) after the merge.